packages/suite/src/views/settings/SettingsDevice/SettingsDevice.tsx

```diff
@@ -89,7 +92,7 @@ export const SettingsDevice = () => {
     const deviceModelInternal = device.features.internal_model;
 
     const supportsDeviceAuthentication = SUPPORTS_DEVICE_AUTHENTICITY_CHECK[deviceModelInternal];
-    const supportsFirmwareRevisionCheck = !supportsDeviceAuthentication; // Older devices with no secure element
+    const supportsFirmwareRevisionCheck = SUPPORTS_FIRMWARE_REVISION_CHECK[deviceModelInternal];
```

Comment:
I could just remove the condition (always true), but I think keeping it granular is no harm done :slightly_smiling_face:

Reply:
Removed in amended commit 1e7ebc1bc5f15dc78ae41aa132c7d3d684dc017b